card/mod/carrierwave/lib/carrier_wave/file_card_uploader.rb

```diff
@@ -1,4 +1,9 @@
 module CarrierWave
+  class << self
+    def tmp_path
+      @tmp_path ||= File.expand_path("tmp", root)
```

Comment:
Should probably be 
```
@tmp_path ||= Card.paths["tmp"].existent.first`
```
That way if someone changes the tmp directory configuration, it will take effect everywhere.